### ISSUE TITLE
there is no training process.

### ISSUE THREAD

**Issue by krystalonly** [SJoJoK/3DGStream#21]
![1](https://github.com/SJoJoK/3DGStream/assets/146855486/9a328793-0010-4183-882b-b48ae48be2f9)
My environment is from gaussian_splatting.
And I use flame_steak in text file directly.

**Comment by liysu** [comment]
I have the same problem. Have you solved it？

**Comment by MarcWangzhiru** [comment]
You need to make your own dataset, only the first frame and trained point cloud are provided in the test folder, you need to extract the subsequent frames yourself and organize them as demo. In fact if you print (frames) you'll get an empty list in train_frames.py.
![屏幕截图 2024-07-11 010733](https://github.com/SJoJoK/3DGStream/assets/114221607/10ee2672-567c-4485-a936-19d1f5bfcd3f)

**Comment by SJoJoK** [comment]
Yeah, just as @MarcWangzhiru said, you need to follow the README and organize the multi-view dataset.

**Comment by xxjcz** [comment]
Hello, how did you organize your dataset? I downloaded the raw dataset for "flame steak," but it consists of a separate video for each viewpoint. I'm not quite sure which frames correspond to the first frame. Could you clarify?
